src/test/ERC721.t.sol

```diff
@@ -490,9 +490,9 @@ contract ERC721Test is TestPlus {
     }
 
     function testTransferFromApproveAll(uint256 id, address to) public {
-        if (to == address(0) || to == address(this)) to = address(0xBEEF);
-
         address from = address(0xABCD);
+        if (to == address(0) || to == address(this) || to == from) to = address(0xBEEF);
```

Comment:
```suggestion
        address from = address(0xABCD);
        
        if (to == address(0) || to == address(this) || to == from) to = address(0xBEEF);
```

Reply:
yep going through all these now - sometimes the fuzzer doesn't catch even with ci profile it seems